.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.idea
```

Comment:
@carllerche Would you be open to leave it ?

I put it there because I used [IntelliJ and the Rust plugin] (https://intellij-rust.github.io) and I figured that others might do it too.